packages/ui/src/components/icons/DiscordIcon.tsx

```diff
@@ -3,7 +3,8 @@ import { cva, VariantProps } from "cva";
 const discordIconStyles = cva("", {
   variants: {
     color: {
-      blurple: "dark:text-blue-400 text-[#5865F2] hover:text-[#7289DA] dark:hover:text-blue-500",
+      blurple:
```

Comment:
Thinking about this, this could work, but we want it to look modern and "live" so I'm not sure if this would be suitable....? No one sees the old google logo really anymore :(